src/Okta.Sdk/DefaultRetryStrategy.cs

```diff
@@ -0,0 +1,158 @@
+﻿// <copyright file="DefaultRetryStrategy.cs" company="Okta, Inc">
+// Copyright (c) 2014 - present Okta, Inc. All rights reserved.
+// Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Okta.Sdk
+{
+    /// <summary>
+    /// Default retry strategy
+    /// </summary>
+    public class DefaultRetryStrategy : IRetryStrategy
+    {
+        /// <summary>
+        /// The default delta used in the back-off formula to account for some clock skew in our service
+        /// </summary>
+        public const int DefaultBackOffSecondsDelta = 1;
```

Comment:
In this variable we capitalize backoff as two separate words, but in the private variable we capitalize it like its all one word. They are both defensible but we should be consistent.

The [only example I see in the .NET framework](https://docs.microsoft.com/en-us/dotnet/api/microsoft.exchange.webservices.data.serverbusyexception.backoffmilliseconds?view=exchange-ews-api#Microsoft_Exchange_WebServices_Data_ServerBusyException_BackOffMilliseconds) uses "BackOff" so maybe we should go with that.